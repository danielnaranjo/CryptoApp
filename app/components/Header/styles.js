import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  header: {
    fontSize: 24,
  },
});

export default styles;
