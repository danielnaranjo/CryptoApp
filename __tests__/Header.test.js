import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../app/components/Header';

it('Header renders correctly', () => {
  const tree = renderer.create(<Header title="TEST" />);
  expect(tree).toMatchSnapshot();
});
