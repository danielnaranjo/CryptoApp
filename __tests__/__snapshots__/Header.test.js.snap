// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontFamily": "Pacifico",
        "fontSize": 44,
      }
    }
  >
    TEST
  </Text>
</View>
`;
